asmcli/tests/lint_variables

```diff
@@ -0,0 +1,41 @@
+main() {
```

Comment:
Let's make sure the bash header is here `#!/usr/bin/env bash`

Reply:
Oops. Done.